client/marklogicClient.ts

```diff
@@ -76,10 +73,10 @@ export class MlClientParameters {
 }
 
 export class MarklogicClient {
-    params: MlClientParameters;
-    ca: string;
+    params: MlClientParameters
+    ca: string
 
-    mldbClient: ml.DatabaseClient;
+    mldbClient: ml.DatabaseClient
```

Comment:
This comment is out of scope of this PR - but in general, I think we should use the term "client" or "databaseClient" when referring to an instance of either the Node or Java client. I know there was a movement early on to use the term "db" to emphasize that MarkLogic is a database - but I don't think that would be intuitive to a developer. The developer knows they're making HTTP calls to MarkLogic, and they're accustomed to using a "client" to make HTTP calls. Or even if they're thinking in terms of "I'm using the Java/Node SDK", they're accustomed to "I will use the client object in this SDK to talk to the server". 

Additionally, the name "DatabaseClient" itself suggests that a good var name is simply "client" (for example, for a class name "DatabaseReplicator", it would not make sense to call that "db" instead of "replicator").

So no action, I'm just pontificating - but I think it's good to rename this stuff when we see it (though not in this PR), assuming it's not part of a public API.

Reply:
There are a TON of style-related things that need to be refactored. Some highlighted by ESLint, some not. I need to create a story to one big style cleanup.